src/EnergyPlus/UnitarySystem.cc

```diff
@@ -6664,10 +6663,7 @@ namespace UnitarySystems {
                         this->m_MSCoolingSpeedRatio[i] = 1.0;
                     }
                     if (this->m_CoolingCoilType_Num == HVAC::Coil_CoolingWaterToAirHPVSEquationFit) {
-                        std::string MultispeedType = "UnitarySystemPerformance:Multispeed";
-                        if (this->m_DesignSpecMSHPIndex == -1) {
-                            std::string MultispeedType = "Fan:SystemModel";
-                        }
+                        std::string MultispeedType = (this->m_DesignSpecMSHPIndex == -1) ? "Fan:SystemModel" : "UnitarySystemPerformance:Multispeed";
```

Comment:
[src/EnergyPlus/UnitarySystem.cc:6667]:(style),[variableScope],The scope of the variable 'MultispeedType' can be reduced.
[src/EnergyPlus/UnitarySystem.cc:6669]:(style),[shadowVariable],Local variable 'MultispeedType' shadows outer variable
[src/EnergyPlus/UnitarySystem.cc:6669]:(style),[unreadVariable],Variable 'MultispeedType' is assigned a value that is never used.